bin/GameIndex.dbf

```diff
@@ -10217,6 +10340,15 @@ Region = PAL-Unk
 Serial = SLES-51547
 Name   = Next Generation Tennis 2003
 Region = PAL-M5
+vuClampMode = 3
```

Comment:
Would be nice to know why it's needed.

Reply:
Next Generation Tennis 2003 patch fixes the sps by rearranging the cop2 instructions, VU clamping is still required to fix SPS in the menus.

Reply:
Like it says in the patch it works in conjunction with the patch